src/vs/workbench/contrib/debug/browser/debugToolBar.ts

```diff
@@ -392,7 +392,7 @@ export function createDisconnectMenuItemAction(action: MenuItemAction, disposabl
 
 // Debug toolbar
 
-const debugViewTitleItems: IDisposable[] = [];
+const debugViewTitleItems = new DisposableStore();
```

Comment:
Small nit: this should be marked as singleton as well because it never gets disposed.